docs/11.1/api/sidebar.js

```diff
@@ -1,6 +1,6 @@
 const fs = require('fs');
 const path = require('path');
-const { getBuildDocsVersion, getLatestVersion } = require('../../.vuepress/helpers');
+const { getEnvDocsVersion, getLatestVersion } = require('../../.vuepress/helpers');
```

Comment:
This would probably be more readable if each imported prop would be in its own line?

Reply:
https://github.com/handsontable/handsontable/pull/9371#discussion_r861015504